docs/api/modules/Bundling-Modules.md

```diff
@@ -0,0 +1,186 @@
+[👈 Return to Overview](../API.md)
+
+# Bundling
+
+Within `one-app` you can bundle your modules once you've made changes to them. The bundled files are then loaded on the client and server.
+
+## Bundling Modules
+
+Generate your module using [generator-one-app-module](https://github.com/americanexpress/one-app-cli/tree/master/packages/generator-one-app-module).
+
+The module will have the following folder structure. Ensure it follows the pattern below for the build files to be generated correctly:
+
+
+Module:
+```
+
+module
+├── README.md
+├── package.json
+├── src
+|   ├── components
+|   |    └── <NameOfComponent.jsx>
+|   |
+|   └── csp.js
```

Comment:
This `csp.js` file is _technically_ only needed in the root module if I remember correctly.

Reply:
it seems to be generated for any module that is generated using the generator, also this applies root module as well - but i could make it clear

Reply:
Yes its only for Root modules. We don't have a `--generate-root-module` option for the generator yet but technically only set a CSP for root module.
https://github.com/americanexpress/one-app/blob/master/docs/api/modules/App-Configuration.md#csp

Reply:
And we now do have that option in the generator since https://github.com/americanexpress/one-app-cli/pull/25 was merged!

Reply:
Thank you, i've removed this.